### PR TITLE
Feature bigboard

### PR DESCRIPTION
**Pull request by stinnux**
Here is an updated PR for Feature Bigboard, based on the current master version. Board Collapsing/Expanding works again (using either 's' key or the menu  entry).

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -22,3 +22,4 @@ data/cache
 !docker/kanboard/config.php
 node_modules
 bower_components
+data/output.csv
```
Comment (fguillot) [inline .gitignore]: Why this file is there?

**Comment by ArmReu** [comment]
Any update on this feature? We'd absolutely love to use it once released!

**Comment by stinnux** [comment]
I am currently quite busy. I will have a look next week.

**Comment by stinnux** [comment]
I have based the PR on the current master version and removed data/output.csv from the .gitignore file. Can you have a look at it again?

**Comment by stinnux** [comment]
I would really love to have this PR merged, so that i can release the Plugin. 

What needs to be done to have this merged?

File: app/Template/task/time_tracking_details.php [hunk 2]
```diff
@@ -15,6 +15,7 @@
         <tr>
             <th class="column-15"><?= $subtask_paginator->order(t('User'), 'username') ?></th>
             <th><?= $subtask_paginator->order(t('Subtask'), 'subtask_title') ?></th>
+            <th class="column-20">t('Comment') ?></th>
```
Comment (fguillot) [inline app/Template/task/time_tracking_details.php]: There is no comment field in this table, where does it come from?

File: app/Template/task/time_tracking_details.php [hunk 3]
```diff
@@ -23,6 +24,7 @@
         <tr>
             <td><?= $this->url->link($this->text->e($record['user_fullname'] ?: $record['username']), 'UserViewController', 'show', array('user_id' => $record['user_id'])) ?></td>
             <td><?= t($record['subtask_title']) ?></td>
+            <td><?= $this->text->markdown($record['comment']) ?></td>
```
Comment (fguillot) [inline app/Template/task/time_tracking_details.php]: Same here

File: assets/js/src/BoardCollapsedMode.js [hunk 4]
```diff
@@ -1,27 +1,57 @@
...
+    }
+};
+
+Kanboard.BoardCollapsedMode.prototype.toggle = function() {
+    var self = this;
+    this.app.showLoadingIcon();
+
+    var url = $('.filter-display-mode:not([style="display: none;"]) a').attr('href');
+
+    if (self.app.hasId("bigboard")) {
```
Comment (fguillot) [inline assets/js/src/BoardCollapsedMode.js]: The Kanboard's code should not include code related to your plugin. I agree to improve this code to make your plugin compatible, but not to hardcode the name of your plugin here.

**Comment by fguillot** [comment]
I'm closing this PR until the comments get addressed.

Note: For your information a lot of changes are happening in the Javascript code.

Thank you for your understanding

**Pull request closed without merging**